### PR TITLE
doc: added '-r' command usage

### LINKED ISSUE #5: Update readme about new `-r` command
See the CLI usage, there's a new `-r` command to create a JSON blueprint from a Github repository. Update the readme about its usage wherever needed.

### PR DESCRIPTION
**Pull request by rodgeraraujo**
Added doc in README of usage the -r command, that generates JSON blueprint from a repository.

Fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +62,13 @@ $ git clone https://github.com/username/repo_name.git && cd repo_name && ~/path/
 $ ./path/to/dgen-rs -g /path/to/json
 ```
 
+# Creates a blueprint form a repository
+
+```bash
+# to generate a JSON blueprint from the repository
+$ ./path/to/dgen-rs -r <username/repo of GitHub repository>
```
Comment (ProCode2) [inline README.md]: Good job! can this be changed to 👇 ? Open to suggestions. I wanted to include Organizations as well.
```suggestion
$ ./path/to/dgen-rs -r <Username or Organization>/<Repository Name>
# example: ./path/to/dgen-rs -r rodgeraraujo/Pokedex
```

**Review by ProCode2 — CHANGES_REQUESTED** [review]
One little tweak and we can merge! Thanks a lot.

File: README.md (hunk 1 above)
Comment (rodgeraraujo) [reply README.md]: No problem, I will change this

**Review by ProCode2 — APPROVED** [review]
Merging! 🚀

**Pull request merged**